app/src/main/java/com/github/se/gomeet/viewmodel/UserViewModel.kt

```diff
@@ -45,7 +45,8 @@ class UserViewModel(val currentUID: String? = null) : ViewModel() {
       email: String,
       phoneNumber: String,
       country: String,
-      pfp: String = ""
+      pfp: String = "",
+      favorites: List<String> = emptyList()
```

Comment:
It doesn't make sense to introduce this here, since upon the creation of a user that user won't have any events. This technically doesn't hurt though and it doesn't change the code functionality, so I'll approve the PR.